doc/code.html

```diff
@@ -302,7 +315,7 @@ <h3 id="Command">Your first program</h3>
 </p>
 
 <pre>
-$ <b>cd $GOPATH/src/github.com/user/hello</b>
+$ <b>cd ~/myproject/hello</b>
 $ <b>git init</b>
 Initialized empty Git repository in /home/user/work/src/github.com/user/hello/.git/
```

Comment:
I think the output of this `git init` command needs to be updated:

```suggestion
Initialized empty Git repository in /home/user/myproject/hello/.git/
```

Reply:
Good catch, I've updated it!